app/mobile-content/pages/shared/page-input/page-input.component.ts

```diff
@@ -0,0 +1,160 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MdDialog, MdDialogRef } from '@angular/material';
+
+import { MobilePage } from '../mobile-page.model';
+import { MobilePageService } from '../mobile-page.service';
+import { MediaService } from '../../../media/shared/media.service';
+import { Medium, mediaType } from '../../../media/shared/medium.model';
+import { SelectMediumDialogComponent } from '../../../media/select-medium-dialog/select-medium-dialog.component';
+import { SelectPageDialogComponent } from '../../select-page-dialog/select-page-dialog.component';
+import { Status } from '../../../shared/status.model';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hip-page-input',
+  styleUrls: ['page-input.component.css'],
+  templateUrl: 'page-input.component.html'
+})
+export class PageInputComponent implements OnInit {
+  @Input() page: MobilePage;
+  audioTitle = '';
+  imageTitle = '';
+  infoPages = new Map<number, MobilePage>();
```

Comment:
Are you sure, that you want a `Map<number, MobilePage>` instead of just an `Array<MobilePage>`? I mean, I don't know what the `number` will be, maybe it's not just an index. In that case, ignore this comment.